library/Claim.class.php

```diff
@@ -1079,6 +1080,12 @@ function box14qualifier()
       return empty($this->billing_options['box_14_date_qual']) ? '431' :
               $this->billing_options['box_14_date_qual'];
   }
+
+  function en_pos_code()
+  {
```

Comment:
Note also that POS code appears in paper claims.  However if you retain the function name facilityPOS then that will be handled and you will also not need to change gen_x12_837.inc.php.